### PR TITLE
Add temporary repo-level README

### PR DESCRIPTION
**Pull request by thompsonmj**
For context to those who come across the repository.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,11 @@
-# ai_explanability
\ No newline at end of file
+# ai_explanability
+
+This repository contains code for 4 projects in the `src/` directory:
+* `ImageomicsButterflies/`
```
Comment (hlapp) [inline README.md:4]:
```suggestion
* [`ImageomicsButterflies/`](src/ImageomicsButterflies/)
```

File: README.md [hunk 2]
```diff
@@ -1 +1,11 @@
...
+* `butterflies_transformation/`
```
Comment (hlapp) [inline README.md:5]:
```suggestion
* [`butterflies_transformation/`](src/butterflies_transformation/)
```

File: README.md [hunk 3]
```diff
@@ -1 +1,11 @@
...
+* `class_cvae/`
```
Comment (hlapp) [inline README.md:6]:
```suggestion
* [`class_cvae/`](src/class_cvae/)
```

File: README.md [hunk 4]
```diff
@@ -1 +1,11 @@
...
+* `sinGAN/`
```
Comment (hlapp) [inline README.md:7]:
```suggestion
* [`sinGAN/`](src/sinGAN/)
```

File: README.md [hunk 5]
```diff
@@ -1 +1,11 @@
-# ai_explanability
\ No newline at end of file
+# ai_explanability
```
Comment (hlapp) [inline README.md:1]: Is there perhaps a better succinct title line than simply repeating the repo name? For example, something even as simple as _AI explainability approaches for Imageomics_.

**Review by hlapp — APPROVED** [review]
See suggestions I included. I won't object if for now there isn't a more descriptive title line, but I'd argue at least it deserves some thought, and I give an example as illustration.

**Comment by thompsonmj** [comment]
Good points! I updated based on those comments.

**Comment by DavidCarlyn** [comment]
Looks good to me. Should help users navigate this repo until we decide to split it apart

**Pull request merged**